src/authextension.ts

```diff
@@ -3,6 +3,7 @@
 import { GlobalState } from './constants';
 import fetch from 'node-fetch';
 import { Config } from './config';
+import { getRedHatUUID } from '@redhat-developer/vscode-redhat-telemetry/lib';
```

Comment:
why /lib?

Reply:
It got added automatically after i used that fucntion.